examples/bitbucket/README.md

```diff
@@ -0,0 +1,43 @@
+## Bitbucket EventListener
+
+Creates an EventListener that listens for Bitbucket webhook events.
+
+### Try it out locally:
+
+1. To create the Bitbucket trigger and all related resources, run:
+
+   ```shell script
```

Comment:
Should be `shell-script`? or just `bash`?
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4989